WcaOnRails/app/assets/stylesheets/wca.scss

```diff
@@ -224,8 +224,10 @@ table.wca-results {
 }
 
 
-// Set flag imitations for competitions taking place in multiple countries.
 .f16.flag {
+  vertical-align: middle;
```

Comment:
Yeah... this is kind of silly that I had to change some css, but my changes affected the competitions search api that is used by the omni search. See my changes to WcaOnRails/vendor/assets/javascripts/jquery.wca-autocomplete.js for more details.